### PR TITLE
fix: remove madara as dep

### PR DESCRIPTION
**Pull request by greged93**
Remove Madara as a dependency of the repo.

Time spent on this PR: 0.1 day

Resolves: #NA

# Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Testing

# What is the new behavior?
Madara is no longer a dependency of the conformance-test-utils crate.

# Does this introduce a breaking change?

- [ ] Yes
- [x] No

# Added changes to [`CHANGELOG.md`](../CHANGELOG.md)

- [x] Yes
- [ ] No

### CODE REVIEW

**Review by ftupas — DISMISSED** [review]
Looks good! Just a comment regarding a duplicate wrapper but we can create an issue for cleanup

File: crates/conformance-test-utils/src/hive/genesis.rs [hunk 1]
```diff
@@ -19,6 +20,33 @@ use crate::madara::utils::{
...
+    pub storage: Vec<(ContractStorageKey, StorageValue)>,
+    pub fee_token_address: ContractAddress,
+    pub seq_addr_updated: bool,
+}
+
+#[derive(Deserialize, Serialize)]
+pub struct ContractClassPath {
+    pub path: String,
+    pub version: u8,
+}
```
[the remark this answers is not in the document]
Comment (greged93) [reply crates/conformance-test-utils/src/hive/genesis.rs:48]: Ah I forgot about it, let me update

**Comment by Eikix** [comment]
nice job

**Review by ftupas — APPROVED** [review]
Awesome job, thanks for removing the duplicate wrapper!

**Pull request merged**